### ISSUE TITLE
Failed to build uwb_plugin

### ISSUE THREAD

**Issue by atinfinity** [jacobrejin/uwb_plugin#1]
I tried to build uwb_plugin. In this time, I used docker container.

```bash
docker run -it ros:humble-perception bash
```

And, I run the following command in docker container.

```bash
apt install libignition-cmake2-dev
apt install libignition-rendering6-dev
apt install libignition-gazebo6-dev
git clone https://github.com/jacobrejin/uwb_plugin.git
cd uwb_plugin/build
cmake ..
```

As a result, it occurs the following error.

```bash
CMake Error at CMakeLists.txt:9 (find_package):
  By not providing "Findignition-rendering6.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "ignition-rendering6", but CMake did not find one.

  Could not find a package configuration file provided by
  "ignition-rendering6" (requested version 6.6.4) with any of the following
  names:

    ignition-rendering6Config.cmake
    ignition-rendering6-config.cmake

  Add the installation prefix of "ignition-rendering6" to CMAKE_PREFIX_PATH
  or set "ignition-rendering6_DIR" to a directory containing one of the above
  files.  If "ignition-rendering6" provides a separate development package or
  SDK, be sure it has been installed.
```

I changed CMakeLists.txt. As a result, I could build uwb_plugin.

```diff
-find_package(ignition-rendering6 6.6.4 EXACT REQUIRED)
+ign_find_package(ignition-rendering6 REQUIRED)
```

**Comment by jacobrejin** [comment]
Are you planning on adding new features to the plugin? If not, please use the provided `libuwb_plugin.so` as is. Just set the Gazebo plugin PATH to the path where the file is located. 

If you want to build the plugin from source, you are missing a crucial step, which is to build the `ignition-rendering` library first as the plugin uses a modified version of the `ignition-rendering` library, which is why it is an EXACT REQUIREMENT. I will add repo and the steps required to build the `ignition-rendering` library and update the same in few days.

**Comment by atinfinity** [comment]
@jacobrejin Thank you for your comment.

> Are you planning on adding new features to the plugin? 

Yes.

> Just set the Gazebo plugin PATH to the path where the file is located.

I think that I need to set `IGN_GAZEBO_SYSTEM_PLUGIN_PATH`. Is my understanding right?

> If you want to build the plugin from source, you are missing a crucial step, which is to build the `ignition-rendering` library first as the plugin uses a modified version of the `ignition-rendering library`, which is why it is an EXACT REQUIREMENT. 

I understood that this project uses `ignition-rendering 6.6.4`.

```cmake
# Find the custom Ignition Rendering library
find_package(ignition-rendering6 6.6.4 EXACT REQUIRED)
```

**Comment by atinfinity** [comment]
@jacobrejin I checked <https://github.com/gazebosim/gz-rendering/tags>. But I could not find `6.6.4`

**Comment by jacobrejin** [comment]
Yes you need to set the plugin path variable IGN_GAZEBO_SYSTEM_PLUGIN_PATH.

You cannot find 6.6.4 as it a custom build of the rendering Library which you will have to build from source. Give me some  time will upload the source code for the rendering Library.
